gef-ui/server/types/view-models/bank-review-date-view-model.ts

```diff
@@ -0,0 +1,10 @@
+import { ViewModelErrors } from './view-model-errors';
+
+export type BankReviewDateViewModel = {
+  dealId: string;
+  facilityId: string;
+  bankReviewDate?: { day: string; month: string; year: string };
+  previousPage: string;
+  status?: string;
+  errors?: ViewModelErrors | null;
```

Comment:
I don't think we need `null` if errors is optional?

Reply:
bump

Reply:
`null` is different to `undefined`, `validationErrorHandler` returns `null`

Reply:
@MarRobSoftwire Noticed we have a duplicate type here, this is the same as `FacilityEndDateViewModel`, could they both be generalised, maybe something like `DateViewModel`? I'm not familiar with other view model type namings

Reply:
`FacilityEndDateViewModel` has `facilityEndDate` whilst this one has `bankReviewDate` - i could commonise with some sort of generic but I think that would just obsfucate the code & tie the two together